dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -442,6 +442,10 @@ warning.tooManySymbolicLinks=CWWKE0096W: Too many symbolic links were encountere
 warning.tooManySymbolicLinks.explanation=Symbolic links are shortcuts that point to other files or directories. This error occurs when the system encounters too many symbolic links in a chain and it is unable to resolve the path.
 warning.tooManySymbolicLinks.useraction=To resolve this issue, check the symbolic links in the path and ensure that they do not create circular references. You might also use the 'readlink' command to inspect individual links in the chain.
 
+warning.javaHomeNotSet=CWWKE0097W: The JAVA_HOME environment variable is not set.
+warning.javaHomeNotSet.explanation=The server script cannot determine the location of the Java installation. This issue does not occur if the JAVA_HOME environment variable is set properly, or if the "java" executable, found by searching the PATH, is located in a complete Java installation. The Java installation location is needed to find version-specific configuration files, such as java9.options. If the Java installation cannot be found, these files will not be loaded, and applications that rely on them might not behave as expected.
```

Comment:
To remove the "will" (future tense), you might change "will not be loaded" to "are not loaded" or "do not load"

This change is a nit. The message is clear even with the "will."